rules/network/discovery_post_exploitation_public_ip_reconnaissance.toml

```diff
@@ -0,0 +1,53 @@
+[metadata]
+creation_date = "2020/09/04"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/09/04"
+
+[rule]
+author = ["Elastic"]
+description = """
+It is common for adversaries to test for Internet access and acquire their public IP address after they have gained
+access to a system. Among others, this has been observed in campaigns leveraging the information stealer, Trickbot. This
+rule identifies domains commonly used by adversaries for post-exploitation IP reconnaissance.
```

Comment:
```suggestion
Identifies domains commonly used by adversaries for post-exploitation IP reconnaissance. It is common for adversaries to test for Internet access and acquire their public IP address after they have gained
access to a system. Among others, this has been observed in campaigns leveraging the information stealer, Trickbot. 
```

Reply:
I think it makes more sense to state up front what the rule identifies. Then you can state how the adversaries use. Re-lint if accepted.